strutil/strutil_test.go

```diff
@@ -279,3 +280,27 @@ func (strutilSuite) TestDeduplicate(c *check.C) {
 		c.Assert(strutil.Deduplicate(t.input), check.DeepEquals, t.output)
 	}
 }
+
+func (strutilSuite) TestWordWrap(c *check.C) {
```

Comment:
Was this test added anew? Or is there an existing test for `wrapGeneric()` or `wrapFlow()` that is now (maybe) no longer needed because `wrapFlow()` is just a tiny wrapper around WordWrap ?

Reply:
Yes, this test is added and did not exist before. A single test existed for wrapFlow that tests for a corner case, which I have not moved, i'll get that one moved to strutil!